config/filter.d/asterisk.conf

```diff
@@ -19,7 +19,7 @@ iso8601 = \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d+[+-]\d{4}
 log_prefix= (?:NOTICE|SECURITY)%(__pid_re)s:?(?:\[C-[\da-f]*\])? \S+:\d*( in \w+:)?
 
 failregex = ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Registration from '[^']*' failed for '<HOST>(:\d+)?' - (Wrong password|Username/auth name mismatch|No matching peer found|Not a local domain|Device does not match ACL|Peer is not supposed to register|ACL error \(permit/deny\)|Not a local domain)$
-            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Call from '[^']*' \(<HOST>:\d+\) to extension '\d+' rejected because extension not found in context 'default'\.$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Call from '[^']*' \(<HOST>:\d+\) to extension '[^']*' rejected because extension not found in context
```

Comment:
Why we should do that? The first non-'* glob will be used to match rest of the string, that contains information used to identify an event, namely "rejected because extension not found...". The second part is not required, because under no circumstances contributes to the quality of the expression.
The expression should be long enough to identify (resp. definitely match) the event. And it should be very precise before `<HOST>` (what is also the case). Because it does all this, no matter what is even more. 

Still again, the shorter the expression, the few vulnerable.
Think about possible extensions of log-entries by maintainer in featured versions (added more details at end of log-entry), by similar strings described the same event (conditional details at end of log-entry), etc. 
Not to mention about performance.